Gopkg.toml

```diff
@@ -25,7 +25,7 @@
 
 [[constraint]]
   name = "github.com/coreos/etcd"
-  version = "v3.2.15"
+  revision = "1b3ac99e8a431b381e633802cc42fe70e663baf5"
```

Comment:
Hi @sbezverk,

Tagging a specific revision in `Gopkg.toml` should almost always be avoided unless overriding a transitive constraint. Additionally, did you specifically need to update etcd for your changes? If not, please remove this change. Otherwise please change this to `branch = "master"` and then run `dep ensure` to regenerate the `Gopkg.lock` file.

Reply:
I have absolutely NO need for etcd, the only reason I nailed this release because when I was using version, the build process was failing.

Reply:
Hi @sbezverk,

Again, I think there must be some confusion between us on the role of `dep` for GoCSI and the SPs generated via `gocsi.sh`.